server/controllers/user.js

```diff
@@ -21,3 +22,36 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+exports.createBoard = asyncHandler(async (req, res) => {
+  const user = await User.findById(req.user.id);
+
+  const newBoard = await Board.create({
+    name: "My board",
+    columns: [
+      {
+        name: "In progress",
+        cards: [],
+        createdAt: Date.now(),
+      },
+      {
+        name: "Completed",
+        cards: [],
+        createdAt: Date.now(),
+      },
+    ],
+    user: user,
```

Comment:
Attaching the user to the board in this way has two problems, one is that it won't detect future updates to the user and the other is that it tries to create a new user document and fails because the email is duplicated. A better approach would be to either have a reference to the User model here in the board or the opposite, have an array of references to the Board model in the user

Reply:
Oh, I see! Thank you.